src/desktop_notifier/base.py

```diff
@@ -444,6 +444,7 @@ async def send(self, notification: Notification) -> None:
                 self._current_notifications.appendleft(notification_to_replace)
             logger.warning("Notification failed", exc_info=True)
         else:
+            logger.debug(f"Notification sent: {notification}")
```

Comment:
nit: I prefer `logger.debug("Notification sent: %s", notification)`, here an in other places, because it does not actually perform any string formatting until a handler is invoked for the log event. In case of DEBUG logs, this means that means almost never in practice.